src/main/java/co/inventorsoft/jdbc/repository/MailDaoImpl.java

```diff
@@ -0,0 +1,98 @@
+package co.inventorsoft.jdbc.repository;
+
+import co.inventorsoft.jdbc.configuration.Driver;
+import co.inventorsoft.jdbc.model.Mail;
+import org.springframework.stereotype.Repository;
+
+import java.io.IOException;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+@Repository
+public class MailDaoImpl implements MailDao {
+
+     private static final String TABLE_MAIL_COLUMN_ID = "id";
+     private static final String TABLE_MAIL_COLUMN_RECIPIENT = "recipient";
+     private static final String TABLE_MAIL_COLUMN_SUBJECT = "subject";
+     private static final String TABLE_MAIL_COLUMN_BODY = "body";
+     private static final String TABLE_NAME_MAIL = "mail";
+    private ResultSet resultSet;
```

Comment:
Format code properly.
Keep 1 empty line after constants and before injected objects.